### DISCUSSION TITLE
Autologin with cluster credentials

### DISCUSSION THREAD

**Discussion by Muscule** [chaos-mesh/chaos-mesh#1883] [Q&A]
Are there any plans to make authorisation with user's credentials like in prometheus or grafana?

**Comment by YangKeao** [comment]
There are multiple options to use a `User` credential:

1. Import the `KUBECONFIG` of the user.
2. Adapt the `User` mechanism of the cloud provider (e.g. GKE...).

I believe these features are on the plan and we will start to develop them right after 2.0 release. Does it fit your needs?

**Reply by Muscule** [reply]
Yes, it does. But 2.0 release we'll not see very soon, I guess. For me it is important to see the exact user, who launches experiments.
